admin_guide/out_of_resource_handling.adoc

```diff
@@ -298,7 +540,7 @@ those pods first.
 
 [NOTE]
 ====
-At this time, volumes that are backed by local disk are only deleted when a pod
+Volumes that are backed by local disk are only deleted when a pod
```

Comment:
When we rev Kubernetes, are the changes implemented in earlier versions of OpenShift, 3.5 and 3.4?